### PR TITLE
Format all the code from ./api with Black

### PR DESCRIPTION
**Pull request by tudoramariei**
Please don't take any of these changes personal (e.g. how to place brackets, `"` instead of `'` etc.).
This should be the standard going forward and a checker is in place to make sure the code is compatible with these changes.

Make your code "Black" with:
`black --line-length 80 --target-version py37 --exclude migrations ./api`
(you can install black using `pip`)

- line length: 80
- target version: Python 3.7

### CODE REVIEW

File: api/buildings/migrations/0001_initial.py [hunk 1]
```diff
@@ -7,43 +7,87 @@ class Migration(migrations.Migration):
```
Comment (aramboi) [inline api/buildings/migrations/0001_initial.py]: Migrations are auto-generated code and should be whitelisted in `black`. We don't want it messing around with these.

Comment (aramboi) [reply api/buildings/migrations/0001_initial.py]:
```
black --line-length 80 --target-version py37 --exclude migrations ./api
```

**Comment by aramboi** [comment]
Most of this stuff was taken care of in https://github.com/code4romania/seismic-risc/pull/87, so we might want to close this. Thanks for the guidelines, I added them as a make command in the other PR.

**Comment by tudoramariei** [comment]
Closing since the code was formatted in #87

**Pull request closed without merging**